tests/mmptest/mmptest.csproj

```diff
@@ -36,6 +36,7 @@
     <PackageReference Include="NUnit" Version="3.12.0" />
     <PackageReference Include="NUnit.ConsoleRunner" Version="3.11.1" />
     <Reference Include="mmp">
+      <Aliases>global,mmp</Aliases>

```

Comment:
why not use `global::` ?

Reply:
Just gave it a spin and it actually doesn't work. The global alias is there by default, so we must have it here when we define it (otherwise, everything defined coming from mmp would need `mmp::` and we don't want that. However, since we have two `Xamarin.Abi` enums (one from mmptest and one from mmp), the one from mmptest actually overshadows the one from mmp so we don't have a way to refer to it without adding an alias. `global::` in our case would just refer to the exact same alias we were already using.